string droneName
string[] arguments
---
bool success
string reason